stylesheets/base.css

```diff
@@ -97,8 +97,6 @@ label[disabled] {
 }
 
 select {
-    -moz-appearance: none;
```

Comment:
Is it safe to delete them?? is it because it's duplicated in `index.css` ?

Reply:
yes, safe! It's a source file that later gets through compiling process and the final outcome that we deliver (/index.css) is having all the prefixes 🎉